apps/site/src/content/design-system/components/button/design/dos-and-donts/content.mdoc

```diff
@@ -10,12 +10,17 @@
 {% layout-area %}
 ![Two buttons vertically stacked with space between them, labelled ‘Save’ and ‘Delete’. This diagram has a tick.](https://res.cloudinary.com/westpac-gel/image/upload/v1603847085/5f98c3ac28dd13161ae76fd8.png)
 
-Do - Prevent errors by keeping buttons a safe distance apart.
+```html
+<p style={{ 'font-size': 14 }}><span style={{ 'font-weight': 'bold', 'color': '#008000'}}>Do</span> - Prevent errors by keeping buttons a safe distance apart.</p>
```

Comment:
minor: I would try to use the classes, `text-success font-bold typography-body-11` I would add them on the safelist to be loaded even tho it doesn't exist on the html. only on markdowns. 

If that doesn't work I would use <strong></strong> then we don't need font-weight I think, also I would insert fontSize: 0.875rem. 

This comment applies for all of below